UnityProject/Assets/Scripts/UI/UI_OxygenAlert.cs

```diff
@@ -10,12 +10,36 @@ public class UI_OxygenAlert : MonoBehaviour {
 
 	public Image img;
 	private Sprite sprite;
+	private float timeWait;
 
-	void Start ()
+	void Start()
 	{
 		img = GetComponent<Image>();
 		sprite = img.sprite;
-		InvokeRepeating("CycleImg", 1f, 1f); //Cycle images every 1 second
+	}
+
+	private void OnEnable()
+	{
+		UpdateManager.Instance.Add(UpdateMe);
+	}
+
+	private void OnDisable()
+	{
+		if (UpdateManager.Instance != null)
+		{
+			UpdateManager.Instance.Remove(UpdateMe);
+		}
+	}
+
+	void UpdateMe()
+	{
+		timeWait += Time.deltaTime;
+		// Cycle images every 1 second
+		if (timeWait > 1f)
```

Comment:
The value that `timeWait` is compared to should either be a constant or a serialized value.
It's easier to understand and modify if there are fewer magic numbers.

Reply:
I've created a constant now.